src/App.tsx

```diff
@@ -17,35 +13,30 @@ export const App = () => (
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavLink
+            to="/"
+            className={({ isActive }) => classNames('navbar-item', {
+              'is-active': isActive,
+            })}
```

Comment:
Use next logic construction
```
 className={classNames({
          [is-active]: isActivet,
        })}
```

Reply:
Hello) Can you write why it's better than my solution?) Just interesting

Reply:
Agree, not actual